spec/intl.html

```diff
@@ -130,8 +130,10 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
         1. Let _opt_ be a new Record.
         1. Let _matcher_ be ? GetOption(_options_, *"localeMatcher"*, *"string"*, « *"lookup"*, *"best fit"* », *"best fit"*).
         1. Set _opt_.[[localeMatcher]] to _matcher_.
-        1. Let _calendar_ be ? GetOption(_options_, *"calendar"*, *"string"*, *undefined*, *undefined*).
+        1. Let _calendar_ be ? Get(_options_, *"calendar"*).
         1. If _calendar_ is not *undefined*, then
+          1. Let _calendar_ be ? ToTemporalCalendar(_calendar_).
+          1. Set _calendar_ to ? ToString(_calendar_).
```

Comment:
any reason not to do this in one step, with nested AO calls? Same below.

Reply:
No reason, just me being unfamiliar with spec language. So you're suggesting this?
```
1. Let _calendar_ be ? ToString(ToTemporalCalendar(_calendar_)).
```

Reply:
Close! (sorry i can’t do multiline suggestions on mobile)
`? ToString(? ToTemporalCalendar(_calendar_)).`

Reply:
Done!